cfApplication.go

```diff
@@ -3,11 +3,14 @@ package main
 // Application represents an interpretation of a runtime Cloud Foundry application. This structure differs in that
 // the information it contains has been processed to simplify its transformation to a Kubernetes manifest using MTA
 type Application struct {
-	Metadata  Metadata          `json:",inline"`
-	Env       map[string]string `json:"env,omitempty"`
-	Services  Services          `json:"services,omitempty"`
-	Processes Processes         `json:"processes,omitempty"`
-
+	Metadata Metadata `json:",inline"`
+	// Env captures the `env` field values in the CF application manifest.
+	Env map[string]string `json:"env,omitempty"`
+	// Services captures the `services` field values in the CF application manifest.
+	Services Services `json:"services,omitempty"`
+	// Processes captures the `processes` field values in the CF application manifest.
+	Processes Processes `json:"processes,omitempty"`
+	// Sidecars captures the `sidecars` field values in the CF application manifest.
 	Sidecars Processes `json:"sidecars,omitempty"`
```

Comment:
is this `Sidecars Processes` ok? Shouldn't it be `Sidecars sidecars`?

Reply:
Should be `Sidecars Sidecars` where type `Sidecars` is a `[]Sidecar`

Reply:
It should give a conflict, let me rebase